### ISSUE TITLE
Populate players array with online player UUIDs

### ISSUE THREAD

**Issue by iGalaxyYT** [Minehut/Meta#59]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

API

**Describe the feature**

 Populate the `players` array with player UUIDs for each server object on **GET /servers** and **GET /server/:id**, currently it's empty.

How it currently looks:
```
...
"players": []
...
```

How it should look:
```
...
"players": ["563815bb492c4883a43bc01060bb2e2a", "20737a76497a4bacbb934693d6ff152a"]
...
```

**How the feature is useful**

This will allow us to access who is currently online any Minehut server.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by recalibrated** [comment]
Players used to hold player usernames, apparently this feature was removed from the API but there was no reason provided as to why. :confused:

**Comment by iGalaxyYT** [comment]
> Players used to hold player usernames, apparently this feature was removed from the API but there was no reason provided as to why. 😕

Yeah, that's why I want it back but using trimmed UUIDs this time
